test/e2e/kubevirt.go

```diff
@@ -1150,7 +1201,14 @@ passwd:
 			}),
 		)
 	})
-	Context("with user defined networks and persistent ips configured", func() {
+	Context("with user defined networks and persistent ips configured", Ordered, func() {
+		AfterAll(func() {
```

Comment:
Can we do this for the fedoracoreos tests too ? so we reduce chances even more, for example if fedoracores imge get bigger in the future running them the other tests may be a problem again, so let's remove both of them.

Reply:
DONE